integration-tests/src/tests/client/features/delegate_action.rs

```diff
@@ -78,3 +86,149 @@ fn reject_valid_meta_tx_in_older_versions() {
         "{status:?}",
     );
 }
+
+/// Take a list of actions and execute them as a meta transaction, check
+/// everything executes successfully, return balance differences for the sender,
+/// relayer, and receiver.
+///
+/// This is a common checker function used by the tests below.
+fn check_meta_tx_execution(
+    node: impl Node,
+    actions: Vec<Action>,
+) -> (FinalExecutionOutcomeView, i128, i128, i128) {
+    let account_id = &node.account_id().unwrap();
+    let node_user = node.user();
+
+    let relayer = account_id.clone();
+    let sender = bob_account();
```

Comment:
consider passing these as arguments -- this will make the test (and the function) more readable.

Reply:
Done. It's a bit more verbose in each test but I can see it is indeed more readable.